src/transforms/lua/v1/mod.rs

```diff
@@ -36,6 +36,9 @@ pub struct LuaConfig {
 
 impl LuaConfig {
     pub fn build(&self) -> crate::Result<Transform> {
+        warn!(
+            "The `lua` transform API version 1 is deprecated. Please convert your script to version 2."
```

Comment:
```suggestion
            "DEPRECATED The `lua` transform API version 1 is deprecated. Please convert your script to version 2."
```

Per: https://github.com/vectordotdev/vector/blob/master/docs/DEPRECATION.md#deprecation